commons/src/main/java/org/frankframework/util/StringUtil.java

```diff
@@ -266,7 +267,7 @@ public static Stream<String> splitToStream(@Nullable final String input) {
 	 *
 	 * @param input the string to split, can be {@literal null}.
 	 * @param delim the delimiters to split the string by
-	 * @return a {@link List} of strings. An empty list if the input was {@literal null}.
+	 * @return a (modifiable) {@link List} of strings. An empty list if the input was {@literal null}.
```

Comment:
Making the result an unmodifiable list would make Sonar a bit happier, but sometimes is a bit of an extra hassle so I decided to leave this as it is and document it.